src/input_parsers/PropertyParser.cpp

```diff
@@ -138,7 +138,7 @@ void PropertyParser::processSingleLine( const String &line, InputQuery &inputQue
 
             auto subToken = subTokens.begin();
             ++subToken;
-            unsigned layerIndex = 2 * atoi( subToken->ascii() ) - 1;
```

Comment:
Yes, this makes sense. It was originally put in to maintain compatibility with the old ws_ properties, now that WS+ReLU layers are stored as two separate layers. Lets just check with @anwu1219 that this doesn't break anything of his.

Reply:
Sorry somehow just saw this message. I think this should be fine on my end. Maybe it'd be good to put the convention that we are treating ws and relu as two separated layer somewhere in the README.

Reply:
Thanks, Andrew.